File: exercism_data/src/135.rb
class Phrase
  def initialize(words)
  	@words = words.downcase
  	@array = @words.split(/\W+/)
  end

  def word_count
    @array.each do |x|
      @counts = Hash.new { |hash, key| hash[key] += 1 }
      @counts[x]
    end
  end

  private

end

#Oh boy is it ugly, but it works.  Please help!

File: assignments/python/wc/src/135.py
def word_count(string):
    split_string_list = string.split()
    word_freq_dict = {}
    for word in split_string_list:
        if not word in word_freq_dict:
            word_freq_dict[word] = 1
        else:
            word_freq_dict[word] += 1
    return word_freq_dict
